apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -669,6 +669,7 @@ function initializeBlocklyWrapper(blocklyInstance: GoogleBlocklyInstance) {
   };
 
   blocklyWrapper.inject = function (container, opt_options) {
+    blocklyWrapper.hasLoadedBlocks = false;
```

Comment:
Why do we need to set both `Blockly.hasLoadedBlocks` in `cdoUtils` and `blocklyWrapper.hasLoadedBlocks` in this wrapper to false to start?

Reply:
My read is that this line sets it as the default state. We toggle it to true after loading blocks. But we also need to set it back to false whenever we might load more blocks onto the workspace (e.g. starting over).

It could be that this line (added early in the commit history) is now redundant, or it could be that we need here because we populate a flyout before actually loading blocks onto the workspace.

Reply:
@mikeharv that is all correct :) I did add the second one later when I realized start over wasn't working. I think we need it in both places to make sure toolbox blocks load correctly (originally that check was on the hidden definition workspace existing, now it is this flag).

Reply:
It could be worth commenting to indicate the two places where it is set serve slightly different purposes, but no strong feelings either way!

Reply:
added clarifying comments!